src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -218,7 +227,8 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
       downloaded_file = None
       if dump_file['recordType'] == 'zone':
         url = str(dump_file['url']).strip()
-        downloaded_file = self._sas.DownloadFile(url, config['serverCert'], config['serverKey'])
+        downloaded_file = self._sas.DownloadFile(url, config['sasTestHarnessCert'],
+                                                 config['sasTestHarnessKey'])
         ppa_zone_data.extend(downloaded_file['recordData'])
```

Comment:
You cannot extend directly due to the format of the file (it is not just a list). I suggest you use the objects in PR #485 to make your life much easier (and it will help us to have consistent code).

Reply:
As FAD   Object PR485 is recently merged to master and made necessary changes to align   with     FAD Object.